src/test/java/org/jenkinsci/plugins/github_branch_source/GithubAppCredentialsContextualizationTest.java

```diff
@@ -0,0 +1,99 @@
+                                + "\"}")));
+
+        // Add wiremock responses for Repository
+        githubApi.stubFor(get(urlEqualTo("/repos/cloudbeers/multibranch-demo"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json; charset=utf-8")
+                        .withBodyFile("../contextualization/body-repos-cloudbeers-multibranch-demo.json")));
+        githubApi.stubFor(get(urlEqualTo("/repos/cloudbeers/multibranch-demo/branches"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json; charset=utf-8")
+                        .withBodyFile("../contextualization/body-repos-cloudbeers-multibranch-demo-branches.json")));
+        githubApi.stubFor(get(urlEqualTo("/repos/cloudbeers/multibranch-demo/contents/?ref=refs%2Fheads%2Fmaster"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json; charset=utf-8")
+                        .withBodyFile("../contextualization/body-repos-cloudbeers-multibranch-demo-contents.json")));
+        githubApi.stubFor(get(urlEqualTo("/repos/cloudbeers/multibranch-demo/pulls?state=open"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json; charset=utf-8")
+                        .withBodyFile("../contextualization/body-repos-cloudbeers-multibranch-demo-pulls.json")));
+    }
+
+    @Test
+    public void ownerMustBeInferedFromRepository() throws Exception {
+        final var store = CredentialsProvider.lookupStores(r.jenkins).iterator().next();
+
+        final var credentials = GitHubApp.createCredentials("myAppCredentialsWithoutOwner");
+        store.addCredentials(Domain.global(), credentials);
+        credentials.setApiUri(githubApi.baseUrl());
+
+        final var scmSource = new GitHubSCMSource("cloudbeers", "multibranch-demo", null, false);
+        scmSource.setTraits(Arrays.asList(new BranchDiscoveryTrait(true, true)));
+        scmSource.setCredentialsId(credentials.getId());
+        scmSource.setApiUri(githubApi.baseUrl());
+
+        final var multiBranchProject = r.jenkins.createProject(WorkflowMultiBranchProject.class, "multibranch-demo");
+        multiBranchProject.getSourcesList().add(new BranchSource(scmSource));
+        multiBranchProject.scheduleBuild2(0, new Action[0]).getFuture().get();
+
+        final var branchProject = multiBranchProject.getItem("master");
+        assertThat(branchProject, notNullValue());
+    }
+
+    private String createTokenExpiration() {
+        // This token will go stale at the soonest allowed time but will not
+        // expire for the duration of the test
+        // Format: 2019-08-10T05:54:58Z
+        return DateTimeFormatter.ISO_INSTANT.format(Instant.ofEpochMilli(new Date(System.currentTimeMillis()
+                                + Duration.ofMinutes(10).toMillis())
```

Comment:
I think you can use `Instant.now().add(Duration.ofMinutes(10))` or similar, to be more idiomatic.

Reply:
https://github.com/jenkinsci/github-branch-source-plugin/pull/804/commits/d7a0bf399a732a417ba188df4b025bf382a120f5